src/bigdecimal.ts

```diff
@@ -2481,6 +2481,12 @@ export class BigDecimal {
      * One example that shows how 2.0 and 2.00 are **not**
      * substitutable for each other under some arithmetic operations
      * are the two expressions:
+     * new BigDecimal("2.0" ).divide(BigDecimal.valueOf(3), HALF_UP) which evaluates to 0.7 and
+     * new BigDecimal("2.00").divide(BigDecimal.valueOf(3), HALF_UP) which evaluates to 0.67.
```

Comment:
This does not look good in the API documentation: 
<img width="834" alt="image" src="https://user-images.githubusercontent.com/32355782/180658913-98bff310-959a-44b6-bfc6-97f8874b6472.png">


We can put the code in backticks and add one or two line breaks